packages/angular/src/lib/legacy/router/page-router-outlet.ts

```diff
@@ -65,6 +66,7 @@ export class PageRouterOutlet implements OnDestroy, RouterOutletContract {
   private isEmptyOutlet: boolean;
   private viewUtil: ViewUtil;
   private frame: Frame;
+  private router = inject(Router);
```

Comment:
This should be naturally injected if made a constructor parameter, eliminating the need for `inject`, right?

Reply:
yes, but the new inject syntax is just too good not to use haha